core/scripts/Voting.js

```diff
@@ -139,9 +157,9 @@ async function fetchCryptoComparePrice(request) {
     "&limit=1",
     "&api_key=" + CC_API_KEY
   ].join("");
-  console.log(`\n    ***** \n Querying with [${stripApiKey(url, CC_API_KEY)}]\n    ****** \n`);
+  // console.log(`\n    ***** \n Querying with [${stripApiKey(url, CC_API_KEY)}]\n    ****** \n`);
```

Comment:
I commented out all logs because I noticed that the AVS tests are the only ones that print to console. The comments seem only to help when debugging and they themselves are not "tests".

Also mentioned this in the PR description, was more meant to open discussion on this.

Reply:
Ahhh, sorry. I should’ve read the description. Thanks for pointing that out. You make a good point. I’ll think more about what to do with the logs.